core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -357,7 +357,7 @@ trait EntityApiService extends UserInfoDirectives {
                   entityServiceConstructor(ctx)
                     .quicksilverMigration(WorkspaceName(workspaceNamespace, workspaceName))
                     .map { migrationResults =>
-                      StatusCodes.OK -> migrationResults
+                      StatusCodes.OK -> Map("entitiesUpdated" -> migrationResults)
```

Comment:
The previous migration returned a map of entityType->numberOfRowsUpdated; this new migration just returns a single sum of numberOfRowsUpdated.